src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
+        $prefix = DIRECTORY_SEPARATOR.'Resources'.DIRECTORY_SEPARATOR.'config'.DIRECTORY_SEPARATOR;
+        $apiResourcesGlob = $prefix.'api_resources.{xml,yml}';
```

Comment:
you should also allow `.yaml`, it's not very common in Sf but a perfectly valid extension nonetheless.